test/verify/check-services

```diff
@@ -188,6 +189,270 @@ WantedBy=default.target
+        m.execute("timedatectl set-time '2020-01-01 16:10:00'")
+        b.wait_present(svc_sel('yearly_timer.timer'))
+        self.assertIn("Fri 2021-01-01 01:22", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("systemctl stop yearly_timer.timer")
+        # creates a new monthly timer that runs on 6th at 14:12 and 8th at 21:12 of every month
+        b.wait_visible("#create-timer")
+        b.click('#create-timer')
+        b.wait_popup("timer-dialog")
+        b.set_val("#servicename", "monthly_timer")
+        b.set_val("#description", "Monthly timer")
+        b.set_val("#command", "/bin/sh -c 'while true; do sleep 5; done'")
+        b.click("#boot-or-specific-time button")
+        b.click("#boot-or-specific-time .dropdown-menu li[value=2] a")
+        b.click("#drop-repeat button")
+        b.click("#drop-repeat .dropdown-menu li[value=44640] a")
+        b.wait_present("[data-content='month-days'][data-index=0]")
+        b.click("[data-content='add'][data-index=0]")
+        b.click("[data-index=0][data-content='month-days'] button")
+        b.click("[data-index=0][data-content='month-days'] .dropdown-menu li[value=6] a")
+        b.set_val("[data-index=0][data-content='hours']", "14")
+        b.set_val("[data-index=0][data-content='minutes']", "12")
+        b.wait_present("[data-content='month-days'][data-index=1]")
+        b.click("[data-index=1][data-content='month-days'] button")
+        b.click("[data-index=1][data-content='month-days'] .dropdown-menu li[value=8] a")
+        b.set_val("[data-index=1][data-content='hours']", "21")
+        b.set_val("[data-index=1][data-content='minutes']", "12")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('monthly_timer.timer'))
+
+        m.execute("timedatectl set-time '2020-01-01 16:15:00'")
+        m.execute("systemctl start monthly_timer.timer")
+        b.wait_present(svc_sel('monthly_timer.timer'))
+        b.wait_in_text(svc_sel('monthly_timer.timer'), "Monthly timer")
+        self.assertIn("Mon 2020-01-06 14:12", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-01-07 00:00:00'")
+        b.wait_present(svc_sel('monthly_timer.timer'))
+        self.assertIn("Wed 2020-01-08 21:12", m.execute("LC_ALL=C systemctl list-timers"))
+        # checks if timer runs on next month February 2020 on same dates
+        m.execute("timedatectl set-time '2020-02-05 00:00:00'")
+        b.wait_present(svc_sel('monthly_timer.timer'))
+        self.assertIn("Thu 2020-02-06 14:12", m.execute("LC_ALL=C systemctl list-timers"))
+        # checks if timer runs on 8th March 2020 at 21:12
+        m.execute("timedatectl set-time '2020-03-07 00:00:00'")
+        b.wait_present(svc_sel('monthly_timer.timer'))
+        self.assertIn("Sun 2020-03-08 21:12", m.execute("LC_ALL=C systemctl list-timers"))
```

Comment:
see above, probably a race condition (maybe it would be good to write a function to cover waiting for this info)